web/src/components/home/subcomponents/song_of_the_week/song_card.js

```diff
@@ -33,7 +30,7 @@ function SongCard(props) {
       </Box>
       <Box sx={songCardStyles.songCardRightContent}>
         <Box sx={songCardStyles.songInfoContainer}>
-          <Typography variant="h6" sx={songCardStyles.songName}>
+          <Typography variant="h5" sx={songCardStyles.songName}>
```

Comment:
Do we also want to make the variants a common named const?

Reply:
I think they kinda are no? also this isn't going to be used again